src/pages/calendar/Day.jsx

```diff
@@ -72,7 +72,7 @@ const Day = ({ time, hasWritten, date, styleConfig, recordable, year }) => {
         const apiUrl = `${import.meta.env.VITE_API_URL}/calendar/memories/${dateInFormat}`;
     
         
-        if (recordable) {
+        if (!isRecordable(year, time)) {
```

Comment:
제가 말씀드린 수정방향은 이쪽이었어요! 
```suggestion
        if (!recordable) {
```